package.json

```diff
@@ -22,10 +22,12 @@
     "@radix-ui/react-scroll-area": "^1.0.5",
     "@radix-ui/react-tabs": "^1.0.4",
     "@storybook/api": "^6.5.9",
-    "@storybook/components-marketing": "^2.3.0",
+    "@storybook/components-marketing": "2.3.1--canary.74.069e59a.0",
     "@storybook/design-system": "7.13.1",
     "@storybook/icons": "^1.2.2",
     "@storybook/theming": "^6.5.9",
+    "auto": "^11.1.1",
```

Comment:
What does `auto` do in this project? There's no releases.

Reply:
Damn. That was a false manip on my side. It was meant to be updated on the icons library .... Too many repos, ...

Reply:
Fixed.